src/libraries/Microsoft.PowerFx.Core/Utils/GetTokensUtils.cs

```diff
@@ -35,10 +35,10 @@ internal static class GetTokensUtils
                     {
                         case BindKind.Control:
                         case BindKind.OptionSet:
-                        case BindKind.PowerFxResolvedObject:
                             tokens[item.Name] = TokenResultType.HostSymbol;
                             break;
                         case BindKind.LambdaField:
+                        case BindKind.PowerFxResolvedObject:
```

Comment:
why this change?

Reply:
This is a rule-scope vs. global name resolver thing. 

We have LSP tests checking these TokenResultTypes.

Before, all the locals where rule-scope, which meant LambdaField, which returns TokenResultType.Variable.

Now, all the locals are NameResolver, which is  BindKind.PowerFxResolvedObject:.

Reply:
Right, that sort of makes me think they should also be HostSymbol token types, since really, they're coming from the host.

Reply:
but there are existing tests that expect these engine globals to be a TokenResultType.Variable.

Reply:
Right, I was being lazy because we were reusing scopes. If we were doing it from scratch, they should be HostSymbol. I'd change the tests. 

All this does is drive formulabar colorizing, for what it's worth.